### ISSUE TITLE
Virus scanner false positives

### ISSUE THREAD

**Issue by Garulf** [Flow-Launcher/Flow.Launcher#2565]
From time to time we get reports of Flow Launcher being detected as potentially malicious software, so I'd like to pin the issue here for any further discussion.

## Whats a false positive?

A false positive occurs when a diagnostic test incorrectly identifies a condition or attribute as present when it is actually not, leading to a mistaken positive result.

## Why does this happen?

Software is complex and virus scanners use many techniques to discover malicious code. Sometimes well meanign code can look malicious to the algorithm a particular scanner is using.

## What can you do?

1. Scanning and scrutinizing software you download from the internet is a great first step!
Flow Launcher is open source and all it's code is accessible. While we understand not everyone has the skills required to read code we suggest any who do to inspect the code and ensure it's safety. 

2. Report any false positives to your virus scanner's company.
Many virus scanner company's have an option to report a scan as a false positive. Doing so helps you and others from having legitimate software get flagged.

3. Make sure you are downloading from one of our official release sources:
    - [GitHub](https://github.com/Flow-Launcher/Flow.Launcher/releases)
    - [Scoop](https://scoop.sh/#/apps?q=flow-launcher)
    - [Chocolatey](https://community.chocolatey.org/packages/flow-launcher)
    - [winget](https://winget.run/pkg/Flow-Launcher/Flow-Launcher)

**Comment by jjw24** [comment]
I don't believe this issue is needed anymore- no security vendors are flagging flow as malicious now.
